app/views/api/asciicasts/show.html.slim

```diff
@@ -19,6 +19,14 @@ javascript:
       if (e.data == 'asciicast:play' || e.data[0] == 'asciicast:play') {
         player.play();
       }
+
+      // reveal.js support
+
+      if (e.data == 'slide:start') {
```

Comment:
I think it's unlikely that there's something else out there which sends `slide:start` to all iframes on the page. I did a quick Google search and only reveal.js popups up, so I wouldn't worry about this.

So yeah, I don't know whether majority of users embedding in reveal.js would want the player to start automatically... I would probably want that, but I can see some people wanting more control.

To make it conditional we could have a new data attribute on embed `<script>` tag, like `data-revealjs` which would be required to enable this integration.